### DISCUSSION TITLE
My student authentication using photo authentication failed

### DISCUSSION THREAD

**Discussion by xlnn** [community/community#19915] [GitHub Education]
My student authentication using photo authentication failed, and the rejection of the authentication was instantaneous.What should I do?Thank you!

**Comment by caioagiani** [comment]
Try again and use other means, attach your student card, dashboard prints (student logged in area)

**Comment by baysideengineer** [comment]
Hello @xlnn, you can try it again and make sure that you meet all requirements. If it fails again I would suggest to [contact](https://support.github.com/contact/education?tags=hubberfy_education) GitHub directly. I am pretty sure that they can help you with this. Good luck.
